src/ts-node.ts

```diff
@@ -130,7 +130,7 @@ const DEFAULT_OPTIONS: Options = {
  */
 export function register (opts?: Options) {
   const cwd = process.cwd()
-  const options = extend(DEFAULT_OPTIONS, { project: cwd }, opts)
+  const options = extend(DEFAULT_OPTIONS, opts)
```

Comment:
You're absolutely right! Sorry I missed that yesterday. In that case, I think it's good 👍